akka-actor/src/main/scala/akka/event/DeadLetterListener.scala

```diff
@@ -26,17 +29,40 @@ class DeadLetterListener extends Actor {
   override def postStop(): Unit =
     eventStream.unsubscribe(self)
 
-  def receive = {
+  def receive: Receive = {
     case DeadLetter(message, snd, rcp) ⇒
       count += 1
       val origin = if (snd eq context.system.deadLetters) "without sender" else s"from $snd"
       val done = maxCount != Int.MaxValue && count >= maxCount
-      val doneMsg = if (done) ", no more dead letters will be logged" else ""
+      val doneMsg = if (done) {
+        if ((reEnableOngoing eq null) && suspendDuration.isFinite()) {
+          import context.dispatcher
+          reEnableOngoing = context.system.scheduler.scheduleOnce(
+            suspendDuration.asInstanceOf[FiniteDuration],
+            self, ReEnableLogging)
+          s", no more dead letters will be logged in next :[$suspendDuration]"
+        } else ", no more dead letters will be logged"
+
+      } else ""
+
       eventStream.publish(Info(rcp.path.toString, rcp.getClass,
         s"Message [${message.getClass.getName}] $origin to $rcp was not delivered. [$count] dead letters encountered$doneMsg. " +
+          s"If this is not an expected behavior, then [$rcp] may terminated unexpectedly, " +
```

Comment:
`$rcp` is already in the log message - I'm not sure if this additional line is useful.